pkg/stores/partition/listprocessor/processor.go

```diff
@@ -99,27 +101,28 @@ const (
 // The subfield is internally represented as a slice, e.g. [metadata, name].
 // The order is represented by prefixing the sort key by '-', e.g. sort=-metadata.name.
 type Sort struct {
-	primaryField   []string
-	secondaryField []string
-	primaryOrder   SortOrder
-	secondaryOrder SortOrder
+	Fields [][]string
+	Orders []SortOrder
 }
 
 // String returns the sort parameters as a query string.
 func (s Sort) String() string {
-	field := ""
-	if s.primaryOrder == DESC {
-		field = "-" + field
-	}
-	field += strings.Join(s.primaryField, ".")
-	if len(s.secondaryField) > 0 {
-		field += ","
-		if s.secondaryOrder == DESC {
-			field += "-"
+	dottableRegex := regexp.MustCompile(`[^a-zA-Z0-9_]`)
```

Comment:
What is the reason to use particular regex was chosen instead of checking for the presence of a dot, like in SafeSplit?

Reply:
I had a choice between either specifying a string that contained something other than an identifier character (alnum or underscore), or specifying a set of non-ident characters (which for k8s labels are `./-`). I figured the alnum characters would be more straightforward. But the name is wrong -- if the term contains a character that is not in that character set, it is NOT dottable. Looking at the unit tests now ...